staging/src/k8s.io/legacy-cloud-providers/gce/gcpcredential/credentialutil.go

```diff
@@ -59,20 +59,20 @@ func ReadURL(url string, client *http.Client, header *http.Header) (body []byte,
 	}
 	defer resp.Body.Close()
 
+	limitedReader := &io.LimitedReader{R: resp.Body, N: maxReadLength}
```

Comment:
`InfoS` can handle bytes display autoly, may be trans this to string is not necessary.